coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/transformers/LinkedInOperatorBasedSqlCallTransformer.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.transformers;
+
+import javax.annotation.Nonnull;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import com.linkedin.coral.common.transformers.OperatorBasedSqlCallTransformer;
+
+import static com.linkedin.coral.trino.rel2trino.CoralToTrinoSqlCallConverter.*;
+
+
+/**
+ * This is a subclass of OperatorBasedSqlCallTransformer which transforms a Coral operator into a Trino operator
+ * on SqlNode layer for the LinkedIn specific functions
+ */
+public class LinkedInOperatorBasedSqlCallTransformer extends OperatorBasedSqlCallTransformer {
+
+  public LinkedInOperatorBasedSqlCallTransformer(@Nonnull String linkedInFuncName, int numOperands,
```

Comment:
we arent using @Nonnull annotation for arguments in a lot of places right now. 
Should we introduce it everywhere / remove it for now?
cc: @wmoustafa

Reply:
Ok to remove

Reply:
Removed.